test/definitions/super-agent/logs/suse155-super-agent.json

```diff
@@ -0,0 +1,36 @@
+{
+    "global_tags": {
+        "owning_team": "virtuoso",
+        "Environment": "development",
+        "Department": "product",
+        "Product": "virtuoso"
+    },
+    "resources": [
+        {
+            "id": "host1",
+            "provider": "aws",
+            "type": "ec2",
+            "size": "t3.nano",
+            "ami_name": "suse-sles-15-sp5-chost-byos-v????????-hvm-ssd-x86_64"
+        }
+    ],
+    "instrumentations": {
+        "resources": [
+            {
+                "id": "nr_super_agent",
+                "resource_ids": [
+                    "host1"
+                ],
+                "provider": "newrelic",
+                "source_repository": "https://github.com/newrelic/open-install-library.git",
+                "deploy_script_path": "test/deploy/linux/newrelic-cli/install-recipe/roles",
+                "params": {
+                    "newrelic_cli_option": "-n logs-integration",
+                    "recipe_content_url": "https://raw.githubusercontent.com/newrelic/open-install-library/main/recipes/newrelic/infrastructure/super-agent/suse.yml",
+                    "validate_output": "New Relic Super Agent\\s+\\(installed\\)",
```

Comment:
`validate_output` should be updated to validate logs integration as below.

`"validate_output": "Logs Integration\\s+\\(installed\\)",`